src/utils/validator_state.py

```diff
@@ -27,6 +28,19 @@ def is_on_exit(validator: Validator) -> bool:
     return int(validator.validator.exit_epoch) != FAR_FUTURE_EPOCH
 
 
+def is_exitable(validator: LidoValidator, last_requested_to_exit_index: int) -> bool:
```

Comment:
I'm not sure this belongs here.
last_requested_to_exit_index - it's our internal thing, this is business logic.
but here we store utils that can be used anywhere
Also we should mention one important thing that last_requested_to_exit_index works only if this validator is holding by node operator which last_requested_to_exit_index is.

Reply:
That's why here is `LidoValidator` type for `validator` arg